### PR TITLE
#1 text.py.example, Info texts

### PR DESCRIPTION
**Pull request by Kazandu**
#1 Renamed text.py to text.py.example, edited .gitignore to include text.py (Also removed one line with a specific chat file

Additional:
Added help comments for cookies, discord webhooks, discord role pings. Added info about the structure of the IP file for IP-Pools, added the info about IP-Range interfaces

### CODE REVIEW

File: const.py.example [hunk 1]
```diff
@@ -28,6 +28,7 @@ ENABLED_MODULES = {
     "telegram": False
 }
 
+#If you dont know how to create a Discord webhook read here: https://support.discord.com/hc/en-us/articles/228383668-Intro-to-Webhooks
```
Comment (lekoOwO) [inline const.py.example]: A white space should follow the `#` sign

File: text.py.example [hunk 2]
```diff
@@ -1,36 +1,38 @@
...
+import utils
+
+def get_private_check_text(status, video_id=None):
+    # import getjson
+    # with getjson.build_req(video_id) as res:
+    #     html = res.read().decode()
+    # info = getjson.get_youtube_video_info(video_id, html)
+    # # Do some if-else using info object
+
+    #Change the Text in return "" to customize the Messages in Discord/Push.
```
Comment (lekoOwO) [inline text.py.example]: Also here

**Review by Kazandu — COMMENTED** [review]
Fixed

**Comment by lekoOwO** [comment]
Please follow the commit naming rule and squash these commits into a single commit, thank you!

**Pull request merged**